packages/google_maps_flutter/google_maps_flutter/pubspec.yaml

```diff
@@ -23,19 +23,19 @@ dependencies:
   google_maps_flutter_android: 
     git:
       url: https://github.com/fulpismo/plugins.git
-      ref: v2.2.1-fork.5.1
+      ref: v2.2.1-fork.6
```

Comment:
the tag name will be **v2.2.1-fork.6**.
i'm bumping these refs so all refs point to the tag (which will contain the code in this branch).

Reply:
Do you think we could have a `cacheClusters` flag or something so we can enable/disable this mechanism on the flutter side? With that, we may be able to add a feature toggle to both protect for any unexpected behavior and also measure the performance improvements we might have after enabling this cache. But I think this is more like a nice to have than a must have

Reply:
nice idea. i think it is a good idea to implement it